### DISCUSSION TITLE
Bug or Feature?

### DISCUSSION THREAD

**Discussion by rsReitzner** [windkh/node-red-contrib-shelly#166] [General]
After Update the follow will not working on ShellyPlugS V2:
msg.payload = {
    method: "Switch.Set",
    parameters : {
        id : 0,
        on : true,
        toggle_after : 2 // optional flip back time in seconds
    }
};
return msg;

**Comment by windkh** [comment]
Can you describe what happens? Any output? Logs?

**Comment by rsReitzner** [comment]
There is no error message. The output is always the status, no matter what I set. All Shellys in the house no longer switch

Von: Karl-Heinz Wind ***@***.***>
Gesendet: Donnerstag, 11. Juli 2024 21:41
An: windkh/node-red-contrib-shelly ***@***.***>
Cc: Seidel Ralf ***@***.***>; Author ***@***.***>
Betreff: Re: [windkh/node-red-contrib-shelly] Bug or Feature? (Discussion #166)


Can you describe what happens? Any output? Logs?

**Comment by rsReitzner** [comment]
Hello Karl-Heinz,
I downgraded to version 10.16.1. Now the Shellys are switching again. By the way: Bist du aus Deutschland? Vereinfacht das ganze 😊

Von: Karl-Heinz Wind ***@***.***>
Gesendet: Donnerstag, 11. Juli 2024 21:41
An: windkh/node-red-contrib-shelly ***@***.***>
Cc: Seidel Ralf ***@***.***>; Author ***@***.***>
Betreff: Re: [windkh/node-red-contrib-shelly] Bug or Feature? (Discussion #166)


Can you describe what happens? Any output? Logs?

**Reply by windkh** [reply]
Ok ich schau mal was da passiert ist. Danke für die info

**Comment by windkh** [comment]
Ich habe da einen riesen Bockmist eingebaut. Mit der nächsten Version sollte es wieder gehen.
Danke!!!

**Comment by windkh** [comment]
V10.28.0
